NEWS.md

```diff
@@ -1,3 +1,5 @@
+# rextendr (development version)
```

Comment:
Nice

Reply:
It is automagically created by `usethis::use_dev_version()` :-D I only called this function and then updated test snapshots :) Automation for the win!